ci-operator/step-registry/gather/gcp-console/gather-gcp-console-commands.sh

```diff
@@ -17,6 +17,18 @@ fi
 
 gcloud config set project "$(jq -r .gcp.projectID "${SHARED_DIR}/metadata.json")"
 
+# while gathering logs from a private cluster, proxy setting is required for connecting cluster
+if test -f "${SHARED_DIR}/proxy-conf.sh"
+then
+	echo "Going to enable client proxy by 'source ${SHARED_DIR}/proxy-conf.sh'"
+	echo "------------------------------"
+	cat "${SHARED_DIR}/proxy-conf.sh"
```

Comment:
The content is a bit sensitive, so it is better not print them out.

Reply:
I don't think the printed info is sensitive (see below). BTW, the main concern for me is, if any further commits, many jobs would be triggered another round and I have to analyze their failures once more. Please advise, thanks! 

FYI https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/origin-ci-test/pr-logs/pull/openshift_release/33459/rehearse-33459-periodic-ci-openshift-verification-tests-master-installer-rehearse-4.12-installer-rehearse-gcp/1589512086084390912/artifacts/installer-rehearse-gcp/gather-gcp-console/build-log.txt
Going to enable client proxy by 'source /tmp/secret/proxy-conf.sh'
------------------------------
export http_proxy=http://XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX@35.238.191.86:3128
export https_proxy=http://XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX@35.238.191.86:3128
export no_proxy="localhost,127.0.0.1"
export HTTP_PROXY=http://XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX@35.238.191.86:3128
export HTTPS_PROXY=http://XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX@35.238.191.86:3128
export NO_PROXY="localhost,127.0.0.1"
------------------------------